Segment-Amplitude.podspec

```diff
@@ -21,6 +21,6 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Pod/Classes/**/*'
 
-  s.dependency 'Analytics', '~> 3.2' # support for iOS 7 +
+  s.dependency 'Analytics', '~> 3.6' # support for iOS 7 +
```

Comment:
Yea I thought it's been a while now since we released analytics-ios. Might be worth getting people onto latest version already.